jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/JKubeConfiguration.java

```diff
@@ -38,6 +39,7 @@ public class JKubeConfiguration implements Serializable {
   private static final long serialVersionUID = 7459084747241070651L;
 
   private JavaProject project;
+  private transient ClusterConfiguration clusterConfiguration;
```

Comment:
Why transient?

Reply:
IDE was showing a warning as ClusterConfiguration is not serializable . I think we do not want to expose this field via Serialization